docs/src/running-validator/validator-reqs.md

```diff
@@ -51,10 +51,14 @@ especially for the case of running staked validators.
 
 Running validator for live clusters (including mainnet-beta) inside Docker is
 not recommended and generally not supported. This is due to concerns of general
-docker's containerzation overhead and resultant performance degradation unless
+Docker's containerzation overhead and resultant performance degradation unless
 specially configured.
 
-We use docker only for development purpose.
+We use Docker only for development purposes. Docker Hub contains images for all
+releases at [solanalabs/solana](https://hub.docker.com/r/solanalabs/solana). We
+also provide a
+[convenience script](https://github.com/solana-labs/solana/blob/master/sdk/docker-solana/localnet.sh)
```

Comment:
I don't think we want to make it too easy to use docker, it's generally not something we want devs to do. Linking to the images sounds good though!

Reply:
That's fair -- how do you propose to give people access to that script?  It'll be tough to run with just the container without knowledge of all the ports required.  I tried using `-P` locally and can confirm that it maps to random ports on your machine, which to me defeats the purpose of making it available for development.  Unless we want the docker images to just be available for pure docker setups, and always have people use `solana-test-validator` locally.

On another note, `solana-test-validator` could use some docs.